### PR TITLE
SONAR-9054 PageDefinition API handles organization scope

### PR DESCRIPTION
**Pull request by teryk**

### CODE REVIEW

**Comment by teryk** [comment]
IT will be updated by greg

File: sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java [hunk 1]
```diff
@@ -158,7 +158,7 @@ public Page build() {
       if (name == null || name.isEmpty()) {
         throw new IllegalArgumentException("Name must be defined and not empty");
```
Comment (julienlancelot) [inline sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java]: Please use checkArgument

File: sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java [hunk 2]
```diff
@@ -158,7 +158,7 @@ public Page build() {
...
       }
-      if (qualifiers.length > 0 && GLOBAL.equals(scope)) {
+      if (qualifiers.length > 0 && !COMPONENT.equals(scope)) {
         throw new IllegalArgumentException(format("The scope must be '%s' when component qualifiers are provided", COMPONENT));
```
Comment (julienlancelot) [inline sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java]: Please use checkArgument

**Comment by gregaubert** [comment]
It's ready on my branch [feature/greg/org-pages-ui](https://github.com/SonarSource/sonarqube/tree/feature/greg/org-pages-ui).
You want me to merge it to your branch or I do another pull request ?

**Comment by teryk** [comment]
if it's reviewed, you can merge on my branch @gregaubert

File: sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java (hunk 2 above)
Comment (teryk) [inline sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java]: it's the API, that's why I don't use checkArgument

File: sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java (hunk 1 above)
Comment (teryk) [inline sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java]: it's the API, that's why I don't use checkArgument

File: sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java (hunk 2 above)
Comment (julienlancelot) [inline sonar-plugin-api/src/main/java/org/sonar/api/web/page/Page.java]: The API has accessed to [guava](https://github.com/SonarSource/sonarqube/blob/master/sonar-plugin-api/pom.xml#L43), so you can use it.

**Comment by gregaubert** [comment]
Ok then I'll do a pull request on your branch and ask for @stas-vilchik review there.

**Pull request merged**